examples/ingress-one-to-many.jsonnet

```diff
@@ -0,0 +1,109 @@
+local ingress(name, namespace, rules) = {
```

Comment:
I think it would be easier for beginners to understand if we didn't use a function here.

Reply:
Yes, sure! Where can I put this function?

Reply:
Just no function, I think we should just in-line the entire ingress object.

Reply:
Since so much time has passed, should I close this PR?